lib/refactor-config.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   , directoryFilter :  null         // not supported yet
   , fileFilter      :  '.js'        // the extension of the file to upgrade
   , dryrun          :  true         // true|false if true no changes will be written to upgraded files
+  , moveStrict      :  false        // true|false if true moves 'use strict;' statement to the top of the file
```

Comment:
Let's make this `true` by default as I can see no case where you wouldn't want this.